src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -202,6 +190,34 @@ public SFArrowResultSet(
     }
   }
 
+  public static void closeRootAllocator(RootAllocator rootAllocator) {
```

Comment:
Same here, this block diff is due to reformatting with `google-java-format`.

Reply:
Maybe you can run google-java-format first and merge that ?